.env.example

```
@@ -20,7 +20,7 @@ DISCORD_PROD_SERVER_ID="528728562780501729853789"
 DISCORD_PROD_VERIFY_CHANNEL_ID="5287378262780501729853789"
 DISCORD_SECRET_TOKEN="FHBAHJABDhbHSUeyi7398.S72ljd.HKBVAfug73gifa-74gfwiyB-BHJBDHV07hAJf83"
 HK_ENV="development"
-INTERNAL_AUTH_KEY="bwgybgidbsi-4784gyfgs-475hhkdsbfs-bwgybgidbsi-4784gyfgs-475hhkdsbfs-bwgybgidbsi-4784gyfgs-475hhkdsbfs-fbnauib4783-bhfbsjfbs"
+SHARED_SECRET="bwgybgidbsi-4784gyfgs-475hhkdsbfs-bwgybgidbsi-4784gyfgs-475hhkdsbfs-bwgybgidbsi-4784gyfgs-475hhkdsbfs-fbnauib4783-bhfbsjfbs"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)